package.json

```diff
@@ -524,7 +596,9 @@
     "cdt-amalgamator": "^0.0.11",
     "cdt-gdb-adapter": "^0.0.23",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "serialport": "11.0.0",
```

Comment:
This dependency should not be needed here as it should be pulled in transitively from cdt-gdb-adapter.

I will release cdt-gdb-adapter with the UART changes merged and then you can revisit this PR.

Reply:
Available now in https://github.com/eclipse-cdt-cloud/cdt-gdb-vscode/pull/93

Reply:
Sounds good - I'll push this change up now

Is there a way to test whether removing this won't cause anything to break/not work? I added it in as a result of what I was observing when I was initially developing, so just want to make sure everything works as expected.

Reply:
Looks good to me - it pulled in the required dependencies and changes.